torch/jit/__init__.py

```diff
@@ -237,7 +237,7 @@ def export_opnames(m):
     """
     return torch._C._export_opnames(m._c)
 
-def _get_trace_graph(f, args=(), kwargs=None, _force_outplace=False,
+def _get_trace_graph(f, args=(), kwargs=None, strict=True, _force_outplace=False,
                      return_inputs=False, _return_inputs_states=False):
```

Comment:
Nothing actionable on this PR, but we'll have to keep an eye on the proliferation of boolean flags. Four boolean flags means 16 different "modes" the code can go through, and it's unlikely that we're testing all 16. I wonder if we can coalesce some of the other flags into one (e.g. an `onnx` flag or something)